### PR TITLE
guest_os.go: pass parameters to apt in update function

### PR DESCRIPTION
**Pull request by KexyBiscuit**
Example: `ciel update-os -- --dry-run systemd=1:241 ppp=2.4.7-3`

### CODE REVIEW

**Comment by Garfield550** [comment]
成为了 Go 开发者

**Comment by KexyBiscuit** [comment]
~我也不想写 Go 啊~
好了别版聊了（

File: guest_os.go [hunk 1]
```diff
@@ -210,7 +210,7 @@ func update() {
 	}
 	d.OK()
 
-	exitStatus, runErr = run(`apt -o Dpkg::Options::="--force-confnew" full-upgrade --autoremove --purge --yes`)
+	exitStatus, runErr = run(`apt -o Dpkg::Options::="--force-confnew" full-upgrade --autoremove --purge --yes ` + strings.Join(flag.Args(), " "))
```
Comment (LionNatsu) [inline guest_os.go]: I forgot what flag.Args() will get. Could you give me some examples?
```
ciel update-os arg1 arg2
=> ["arg1", "arg2"]
```
Am I correct?

Comment (LionNatsu) [reply guest_os.go]: Also, how will the following cmdline be parsed?
```
ciel update-os -C "DIRECTORY" --dry-run --print-uris
```

Comment (KexyBiscuit) [reply guest_os.go]: > Also, how will the following cmdline be parsed?
> 
> ```
> ciel update-os -C "DIRECTORY" --dry-run --print-uris
> ```

All parameters passed to `apt*` should be appended after `--`. Example in PR updated.

**Comment by LionNatsu** [comment]
LGTL

**Pull request merged**